sonar-scala-plugin/src/main/resources/org/sonar/l10n/scala/rules/scala/Sonar_way_profile.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "Sonar way",
+  "ruleKeys": [
+    "ParsingError"
```

Comment:
It means it will be activated in SonarLint: is that what we want?

Reply:
no